setup.cfg

```diff
@@ -101,3 +101,6 @@ ignore_missing_imports = True
 
 [mypy-importlib_metadata.*]
 ignore_missing_imports = True
+
+[mypy-pytest.*]
+ignore_missing_imports = True
```

Comment:
If we were checking tests (we're not), you would add pytest in requirements/typing.in, not ignore it.